solr/core/src/java/org/apache/solr/packagemanager/RepositoryManager.java

```diff
@@ -71,15 +71,16 @@ public RepositoryManager(HttpSolrClient solrClient, PackageManager packageManage
     this.solrClient = solrClient;
   }
 
-  public List<SolrPackage> getPackages() {
-    List<SolrPackage> list = new ArrayList<>(getPackagesMap().values());
+  public List<org.apache.solr.packagemanager.SolrPackage> getPackages() {
```

Comment:
oh wow...    I see the problem with SolrPackage...   Is it a Bean?  Is it a class?    Too bad we don't have `o.a.s.client.solrj.request.beans.SolrPackageBean` as a existing naming pattern...   Was taht a thing at one time?